### DISCUSSION TITLE
如何让"NOT_FOUND_DESCRIPTION"这个配置起作用？

### DISCUSSION THREAD

**Discussion by WaterCalm** [apiflask/apiflask#224] [Q&A] [answered]
APIFlask 版本 为 0.12.0
参照文档 (https://apiflask.com/configuration/#not_found_description) 里的说明，修改了
`app.config['NOT_FOUND_DESCRIPTION'] = 'Missing'`

但访问未定义的路由时，提示的描述依旧显示是“Not Found”
```
{
    "detail": {},
    "message": "Not Found"
}
```

或者说这个配置不是影响的返回的响应里的json字符串？

谢谢

**Comment by greyli** [comment, marked as answer]
`NOT_FOUND_DESCRIPTION` 用来设置 OpenAPI spec 里 404 响应的描述。
